java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java

```diff
@@ -257,30 +257,35 @@ private void signTxTest() throws Exception {
         byte[] proto = base64().decode(request);
 
         CommandRequest commandRequest = CommandRequest.parseFrom(proto);
-      if (filePathInJAR.contains("negative")) {
-          try {
-              CommandResponse response =
-                      CommandHandler.dispatch(this, new InternalCommandConnector(hostname, port),
-                              commandRequest);
-          } catch (Exception e) {
-              // A list of negative test cases can be added here.
-              if (
-                      filePathInJAR.contains("bad_qrsignature") && e.toString().contains("QRSIG_CHECK_FAILED")
-              ) {
-                  System.out.println("testcase " + filePathInJAR + " : OK");
-                  ok_valid++;
-              } else {
-                  System.out.println("testcase " + filePathInJAR + " : FAIL");
-                  fail_valid++;
-              }
-              continue;
-          }
-          fail_valid++;
-          System.out.println("testcase " + filePathInJAR + " : FAIL");
-          continue;
+        // If the filename for the test vector has a "negative" then it's
```

Comment:
The big diff here is due to formatting change.

The only changes here are:
1. Comment update.
2. ```filePathInJAR.contains("required_fields_not_present") && e.toString().contains("REQUIRED_FIELDS_NOT_PRESENT")```

Reply:
nit: eventually we'd like to simplify all this logic to something like
```
expected = ...
actual = ...
if (expected != actual) ...
```